src/interface_r/cran-comments.md

```diff
@@ -0,0 +1,23 @@
+## Test environments
+* local OS X install, R 3.4.3
+* ubuntu 12.04 (on Jenkins), R 3.4.3
+* win-builder (devel and release)
+
+## R CMD check results
+
+0 errors | 0 warnings | 1 note
```

Comment:
Whats the note?

Reply:
Related to new release. See the comment below "This is a new release" that addresses the note. This was auto-generated by devtools. This is assuming https://github.com/h2oai/h2o4gpu/issues/477 is resolved and Erin's vignette PR is merged.